bagit.py

```diff
@@ -1219,19 +1226,47 @@ def _parse_tags(tag_file):
         yield (tag_name, tag_value.strip())
 
 
-def _make_tag_file(bag_info_path, bag_info):
+def _make_tag_file(bag_info_path, bag_info, tag_wrap_column):
     headers = sorted(bag_info.keys())
     with open_text_file(bag_info_path, "w") as f:
         for h in headers:
             values = bag_info[h]
             if not isinstance(values, list):
                 values = [values]
             for txt in values:
-                # strip CR, LF and CRLF so they don't mess up the tag file
-                txt = re.sub(r"\n|\r|(\r\n)", "", force_unicode(txt))
+                txt = _format_tag_value(h, txt, tag_wrap_column)
                 f.write("%s: %s\n" % (h, txt))
 
 
+def _format_tag_value(header, txt, tag_wrap_column):
+    txt = force_unicode(txt)
+    if tag_wrap_column > 1:
```

Comment:
Shouldn't this be `if text_wrap_column:` because all of the documentation specifies setting a non-zero value?